packages/algob/src/lib/tx.ts

```diff
@@ -266,16 +268,68 @@ export async function makeAndSignTx (
   return [txns, signedTxn];
 }
 
+/**
+ * Signs transaction object(s) and returns raw signed transaction
+ * Note: Sign transaction is used to sign single transaction and `signTransactionObject` takes
+ * SDK transaction object, signer and signs it.
+ * @param txn Transaction object(s)
+ * @param signer signer
+ */
+export function signTransactionObject (
+  txns: Transaction | Transaction[], signer: wtypes.Signer | wtypes.Signer[]
```

Comment:
This function is too much overloaded and it's not good to return a union type. It doesn't make sense to use this function with single transaction and single signer (because in that case we should use `signTransaction` directly). So I propose to update this function to:
```
export function signTransactions(txns:  Transaction[], signer: wtypes.Signer | wtypes.Signer[]): Uint8Array[]
```

Reply:
updated merged both the types. `TransactionAndSign` in one